README.md

```diff
@@ -10,7 +10,7 @@
 MQTT.js is a client library for the [MQTT](http://mqtt.org/) protocol, written
 in JavaScript for node.js and the browser.
 
-> MQTT [5.0.0-beta.0](https://github.com/mqttjs/MQTT.js/releases/tag/v5.0.0-beta.0) is now available! Try it out and give us feedback! `npm i mqtt@5.0.0-beta.0`
+> MQTT [5.0.0 BETA](https://www.npmjs.com/package/mqtt/v/beta) is now available! Try it out and give us feedback! `npm i mqtt@beta`
```

Comment:
Maybe we should have a specific place to submit feedback? Perhaps a pinned issue where people can put comments

Reply:
https://github.com/mqttjs/MQTT.js/issues/1639